### ISSUE TITLE
support the semantics of vscale

### ISSUE THREAD

**Issue by vfdff** [AliveToolkit/alive2#923]
test: https://alive2.llvm.org/ce/z/JBEHeZ

**%vscale = call i64 @llvm.vscale.i64() nofree willreturn memory(none)** imply the alive2 thinks of it as a common user define function ?
In fact, the vscale is widely used in SVE (Scalable Vector Extension).

**Comment by nunoplopes** [comment]
Yep, vscale is not supported. There's no plan to support it in the short-term.
